spec/core/ics-002-client-semantics/README.md

```diff
@@ -262,7 +262,7 @@ for the given parent `ClientMessage` and the list of network messages.
 The validity predicate is defined as:
 
 ```typescript
-type VerifyClientMessage = (ClientMessage) => Void
+type VerifyClientMessage = (ClientMessage) => bool
 ```
 
 `VerifyClientMessage` MUST throw an exception if the provided ClientMessage was not valid.
```

Comment:
I was also not completely sure about this line, but I left it like this because I saw that in line 283 we currently have `checkForMisbehaviour MUST throw an exception if the provided proof of misbehaviour was not valid.` and `checkForMisbehaviour` is also a validity predicate... Maybe both statements are inconsistent then?

Reply:
@AdityaSripal may have a better understanding of this.